src/test/scala/edu/ie3/simona/model/grid/RefSystemSpec.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * © 2020. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.simona.model.grid
+
+import edu.ie3.util.scala.quantities.Sq
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import squants.{Dimensionless, Each}
+import squants.electro.{Amperes, ElectricPotential, Kilovolts, Ohms, Volts}
+import squants.energy.{Megawatts, Power, Watts}
+
+class RefSystemSpec extends AnyFlatSpec with Matchers {
+
+  "A RefSystem with nominal power and nominal voltage" should "provide corresponding nominal current and nominal impedance" in {
+
+    val nominalPower = 600000
+    val nominalVoltage = 10000
+
+    val nominalPowerAsSquant: Power = Sq.create(nominalPower, Watts)
+    val nominalVoltageAsSquant: ElectricPotential =
+      Sq.create(nominalVoltage, Volts)
```

Comment:
You can simplify this by directly calling squants with Watts or Kilowatts / Volts